src/util/pluginUtils.js

```diff
@@ -269,27 +296,44 @@ export function asLookupValue(modifier, lookup = {}) {
 let typeMap = {
   any: asValue,
   color: asColor,
+  url: asURL,
   angle: asAngle,
   length: asLength,
+  size: asLength,
+  position: asLength,
   lookup: asLookupValue,
+  family: asFontFamily,
+  weight: asFontWeight,
 }
 
+let supportedTypes = Object.keys(typeMap)
+
 function splitAtFirst(input, delim) {
-  return (([first, ...rest]) => [first, rest.join(delim)])(input.split(delim))
+  let idx = input.indexOf(delim)
+  if (idx === -1) return [undefined, input]
+  return [input.slice(0, idx), input.slice(idx + 1)]
```

Comment:
Changed this a little, because I wanted to get the `[typehint, rest]` and if there is no typehint, I wanted `[undefined, rest]`. The previous code would have generated `[rest]` for the last case.